### ISSUE TITLE
CI failures/issues on current master

### ISSUE THREAD

**Issue by AlexDBlack** [deeplearning4j/deeplearning4j#8748]
https://jenkins.konduit.ai/blue/organizations/jenkins/konduitai%2Fdeeplearning4j/detail/PR-284/2/pipeline/

linux-ppc64le-cpu
```
[2020-03-03T22:54:01.306Z] Tests in error: 
[2020-03-03T22:54:01.306Z]   testTransform[0: backend(org.nd4j.linalg.cpu.nativecpu.CpuBackend)={1}](org.nd4j.linalg.dataset.NormalizerStandardizeLabelsTest): test timed out after 30000 milliseconds
```

linux-x86_64-cuda-10.0
```

[2020-03-03T22:51:56.647Z]   EmbeddingLayerTest.testEmbeddingWeightInit:774 expected: org.nd4j.linalg.jcublas.JCublasNDArray<[[    0.0324,   -0.0465,   -0.0584,  ...   -0.2255,    0.1262,    0.0089]]> but was: org.nd4j.linalg.jcublas.JCublasNDArray<[[    0.0324,   -0.0465,   -0.0584,  ...   -0.2255,    0.1262,    0.0089]]>
```

macos-x86_64-cpu-avx2
```
Failed tests:   testRandomBinomial[0: backend(org.nd4j.linalg.cpu.nativecpu.CpuBackend)={1}](org.nd4j.linalg.api.rng.RngTests)
```

windows-x86_64-cuda-10.0
* JVM crash - unclear what test this is (must not extend BaseNd4jTest as test name is not logged before starting - which is another issue in and of itself...

windows-x86_64-cuda-10.1
looks like libnd4j deadlock - look at timestamps here - stuck for an hour in `OneOffTests.test_conv2d_nhwc_failed_1`
```
[2020-03-03T22:25:56.614Z] [ RUN      ] OneOffTests.test_assert_scalar_float32_2
[2020-03-03T22:25:56.614Z] [       OK ] OneOffTests.test_assert_scalar_float32_2 (1 ms)
[2020-03-03T22:25:56.614Z] [ RUN      ] OneOffTests.test_pad_1D_1
[2020-03-03T22:25:56.614Z] [       OK ] OneOffTests.test_pad_1D_1 (1 ms)
[2020-03-03T22:25:56.614Z] [ RUN      ] OneOffTests.test_conv2d_nhwc_failed_1
[2020-03-03T23:29:05.295Z] Sending interrupt signal to process
[2020-03-03T23:29:11.675Z] Sending interrupt signal to process
[2020-03-03T23:29:19.890Z] script returned exit code 127
```

**Comment by AlexDBlack** [comment]
Everything except a windows CUDA issue here is fixed. Will open a new issue for that one.
